test/browser/spec/ol/image.test.js

```diff
@@ -1,85 +1,174 @@
+      });
+      instance.load();
+    });
   });
+  describe('HTML Image loading', function () {
+    let handleLoad, handleError, img;
 
-  it('handles load event', function (done) {
-    img.src = 'spec/ol/data/dot.png';
-    listenImage(img, handleLoad, handleError);
+    beforeEach(function () {
+      handleLoad = sinon.spy();
+      handleError = sinon.spy();
+      img = new Image();
+    });
 
-    setTimeout(function () {
-      expect(handleLoad.called).to.be(true);
-      expect(handleError.called).to.be(false);
-      done();
-    }, 200);
-  });
+    it('handles load event', function (done) {
+      img.src = 'spec/ol/data/dot.png';
+      listenImage(img, handleLoad, handleError);
 
-  it('handles load event when src is set later', function (done) {
-    listenImage(img, handleLoad, handleError);
-    img.src = 'spec/ol/data/dot.png';
+      setTimeout(function () {
+        expect(handleLoad.called).to.be(true);
+        expect(handleError.called).to.be(false);
+        done();
+      }, 200);
+    });
 
-    setTimeout(function () {
-      expect(handleLoad.called).to.be(true);
-      expect(handleError.called).to.be(false);
-      done();
-    }, 200);
-  });
+    it('handles load event when src is set later', function (done) {
+      listenImage(img, handleLoad, handleError);
+      img.src = 'spec/ol/data/dot.png';
 
-  it('handles error event', function (done) {
-    img.src = 'invalid.jpeg';
-    listenImage(img, handleLoad, handleError);
+      setTimeout(function () {
+        expect(handleLoad.called).to.be(true);
+        expect(handleError.called).to.be(false);
+        done();
+      }, 200);
+    });
 
-    setTimeout(function () {
-      expect(handleLoad.called).to.be(false);
-      expect(handleError.called).to.be(true);
-      done();
-    }, 500);
-  });
+    it('handles error event', function (done) {
+      img.src = 'invalid.jpeg';
+      listenImage(img, handleLoad, handleError);
 
-  it('handles cancelation', function (done) {
-    img.src = 'spec/ol/data/dot.png';
-    listenImage(img, handleLoad, handleError)();
+      setTimeout(function () {
+        expect(handleLoad.called).to.be(false);
+        expect(handleError.called).to.be(true);
+        done();
+      }, 500);
+    });
 
-    setTimeout(function () {
-      expect(handleLoad.called).to.be(false);
-      expect(handleError.called).to.be(false);
-      done();
-    }, 200);
-  });
-});
+    it('handles cancelation', function (done) {
+      img.src = 'spec/ol/data/dot.png';
+      listenImage(img, handleLoad, handleError)();
 
-describe('getImage() with context', function () {
-  let image;
-  this.beforeEach(function (done) {
-    image = new ImageWrapper(
-      [0, 0, 1, 1],
-      1,
-      1,
-      'spec/ol/data/dot.png',
-      null,
-      defaultImageLoadFunction,
-      createCanvasContext2D()
-    );
-    image.addEventListener('change', function () {
-      if (image.getState() === ImageState.LOADED) {
+      setTimeout(function () {
```

Comment:
Why was `setTimeout()` used here?

Reply:
This is just because the tests have moved and the diff is hard to read - this test already existed and had the same `setTimeout()`.